src/caller.jl

```diff
@@ -192,22 +197,23 @@ function prepare_argument(x::sresolution)
 end
 
 function prepare_argument(x::Any)
+    in(:ptr, fieldnames(typeof(x))) || error("unrecognized argument")
```

Comment:
Not a change request, just wondering: is there any reason not to write this in infix notation, i.e.
```suggestion
    :ptr in fieldnames(typeof(x))) || error("unrecognized argument")
```

Reply:
There is no reason except that I do not know/do not want to know the precedences: is `a in b || c` the same as `(a in b) || c` or `a in (b || c)`? :)

Reply:
Even with parenthesis it is one comma less!

Reply:
so I just checked it out. Just barely works!
```
julia> Base.operator_precedence(:+)
11
julia> Base.operator_precedence(:in)
7
julia> Base.operator_precedence(:*)
12
julia> Base.operator_precedence(:||)
5
```

Reply:
It is not about the number of characters, but rather readability: `a in b` to me is easier to grok than `in(a,b)`, same for`isa`